Dockerfile-python3

```diff
@@ -0,0 +1,10 @@
+FROM tuttleofx/tuttleofx-env:python3-latest
```

Comment:
Including the python minor version is maybe clearer because it depends on the OS distrib image. Don't you think ?